usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -195,6 +195,7 @@ List of table columns
 + `treasury_account_components` (optional, array[TreasuryAccountComponentsObject], fixed-type)
 + `object_class` (optional, array[string])
 + `program_activity` (optional, array[number])
++ `def_codes` (optional, array[DEFC], fixed-type)
```

Comment:
It looks like the frontend is expecting the `require` / `exclude` pattern used for NAICS, PSC, TAS. cc @MaxwellKendall

Reply:
True enough! @emily-brents this is a trivial change on the front end, and IIRC we acknowledged that it wasn't actually necessary to provide an `excluded` array for this end point given the shape of the data at the moment.

Let me know what you think and we can conform to just seing y'all an array.

Reply:
Yeah, I remember we talked about it and I thought we were just using an array. There's no hierarchy to DEF codes so there's no need to have include/exclude, os a plain array makes the most sense